docs/content/guides/authn/oidc_first_party_auth.adoc

```diff
@@ -27,22 +27,21 @@ There are also some further endpoints, which you'll learn about during the setup
 
 Technically, the setup includes:
 
-* https://hub.docker.com/r/traefik/whoami/[traefik/whoami] - A service that echoes back everything it receives, simulating our main service with endpoints mentioned above.
+* https://hub.docker.com/r/traefik/whoami/[traefik/whoami] - A service that echoes back everything it receives, simulating our main service on the endpoint defined above.
 * https://www.keycloak.org/[Keycloak] - Our identity provider.
-* https://oauth2-proxy.github.io/oauth2-proxy/[OAuth2-Proxy] - Handles the Authorization Code Grant flow for the actual user login.
-* heimdall - Manages everything to enforce the requirements outlined above.
+* https://oauth2-proxy.github.io/oauth2-proxy/[OAuth2-Proxy] - Handles the https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow[Authorization Code Grant flow] for the actual user login.
```

Comment:
Let us rather use https://tools.ietf.org/html/rfc6749#section-4.1 instead of the auth0 link please.